src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -24735,19 +24775,20 @@ static void F(C c, IOut<object> x, IOut<object?> y)
 }";
             var comp = CreateCompilation(new[] { source, NonNullTypesTrue, NonNullTypesAttributesDefinition });
             // PROTOTYPE(NullableReferenceTypes): Report WRN_NullabilityMismatchInAssignment for compound assignment.
```

Comment:
Is the PROTOTYPE comment still applicable?
Btw, it's really nice that this refactoring naturally fixed a number of PROTOTYPE issues :-)